assets/scss/main.scss

```diff
@@ -37,19 +36,17 @@ a {
   opacity: 0;
 }
 
-.leftside-wrapper {
-  flex: 1;
-}
-
 .aside-right {
+  padding: 20px;
   flex-basis: 320px;
-  margin: 0;
-  margin-left: $default-gutters-width;
+}
+
+.main-content {
+  padding: 20px 0 0 20px;
```

Comment:
Why have you changed scss variables to fixed pixels in spacing?

Reply:
2. IMO now it's better. If it's a problem I can easilly chnage that.

Reply:
@zrebcu411 One comment more to 2.
Please let it stay like it is in this PR. My PR will improve it and will fix the issue.

Reply:
Aside part should be definitely wider. Look at onet.pl :)